app/Http/Controllers/Admin/MinistryController.php

```diff
@@ -105,7 +105,8 @@ public function editMinistry(Request $request, $id)
         }
     }
 
-    public function deleteMinistry($id){
+    public function deleteMinistry($id)
+    {
         $delete = Ministry::where('id', $id)->delete();
         if ($delete) {
             echo ("<script>alert(' Ministry details deleted successfully');
```

Comment:
Whoever wrote this code ehn, make them comot this script

Reply:
he will -